rest-api-spec/src/main/resources/rest-api-spec/api/profiling.flamegraph.json

```diff
@@ -0,0 +1,27 @@
+{
+  "profiling.flamegraph":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/observability/current/universal-profiling.html",
+      "description":"Extracts a UI-optimized structure to render flamegraphs from Universal Profiling."
+    },
+    "stability":"stable",
+    "visibility":"private",
+    "headers":{
+      "accept": ["application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_profiling/flamegraph",
+          "methods":[
+            "POST"
+          ]
+        }
+      ]
+    },
+    "body":{
+      "description":"The filter conditions for stacktraces",
```

Comment:
While the sentence is technically correct, it is not necessarily obvious that flamegraph filters are also stacktraces filters.
IMO, it would be less confusing to say "The filter conditions for the flamegraph aggregation".

Reply:
Thanks for the suggestion; I've applied this in https://github.com/elastic/elasticsearch/pull/103675/commits/ed1d3e95509e006f80c76f2991caa183ab9f052d.